docs/docs/topics/ppl.md

```diff
@@ -0,0 +1,170 @@
+---
+title: Pomerium Policy Language
+description: >-
+  This article covers Pomerium Policy Language, used to define secure access policies for routes.
+---
+
+# Pomerium Policy Language
+
+Pomerium Policy Language (**PPL**) is a yaml-based notation for creating easy and flexible authorization policies. This document covers the usage of PPL and provides several example policies.
+
+## At a Glance
+
+Each PPL policy has at the top level a set of `allow` or `deny` actions, with a list of logical operators, criteria, matchers, and values underneath. For example:
+
+```yaml
+allow:
+  and:
+  - domain: 
+      is: example.com
+  - groups: 
+      has: admin
+deny:
+  or:
+  - user: 
+      is: user1@example.com
+  - user: 
+      is: user2@example.com
+```
+
+This policy will allow a user with an email address at `example.com` who **is also** a member of the `admin` group. It will deny `user1` or `user2`, regardless of their domain and group membership.
```

Comment:
```suggestion
This policy will allow a user with an email address at `example.com` who **is also** a member of the `admin` group. It will deny `user1` **or** `user2`, regardless of their domain and group membership.
```